From 60e52ccab7c1f273e8a93380fec5b5fc310600d6 Mon Sep 17 00:00:00 2001
From: c4pp4
Date: Wed, 13 Jul 2022 15:30:21 +0200
Subject: [PATCH 1/1] Add nemo-persist key.

The Launcher's quicklist bookmark plugin only works if the main
application is running.
Bug-Ubuntu: https://launchpad.net/bugs/1720594

Signed-off-by: c4pp4
---
 libnemo-private/org.nemo.gschema.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libnemo-private/org.nemo.gschema.xml b/libnemo-private/org.nemo.gschema.xml
index 207357e..58d4ff3 100644
--- a/libnemo-private/org.nemo.gschema.xml
+++ b/libnemo-private/org.nemo.gschema.xml
@@ -534,6 +534,11 @@
       <summary>Deprecated: Allow Nemo to manage the desktop</summary>
       <description>Deprecated: If this is set to true, Nemo will autostart and manage the desktop</description>
     </key>
+    <key name="nemo-persist" type="b">
+      <default>false</default>
+      <summary>Hold Nemo running seamlessly</summary>
+      <description>If this is set to true, Nemo will autostart seamlessly and manage Unity Launcher's quicklist bookmark plugin. Bug-Ubuntu: https://launchpad.net/bugs/1720594</description>
+    </key>
     <key name="use-desktop-grid" type="b">
       <default>true</default>
       <summary>Which desktop view type to use</summary>
-- 
2.35.1

